docs/jupyter.rst

```diff
@@ -324,26 +320,25 @@ And then create the tileset and track, as before.
 
 .. code-block:: python
 
-    import higlass
-    import higlass.tilesets as hgti
-    import higlass.client as hgcl
-
-    ts = hgti.dfpoints(df, x_col='x', y_col='y')
-
-    (display, server, viewconf) = higlass.display([
-        hgcl.View([
-            hgcl.Track('left-axis'),
-            hgcl.Track('top-axis'),
-            hgcl.Track('osm-tiles', position='center'),
-            hgcl.Track('labelled-points-track', 
-                       tileset=ts,
-                       position='center',
-                       height=600,
-                       options={
-                            'xField': 'x',
-                            'yField': 'y',
-                            'labelField': 'names'
-                        }),
+    from higlass.client import View, Track
+    import higlass.tilesets
+
+    ts = higlass.tilesets.dfpoints(df, x_col='x', y_col='y')
+
+    display, server, viewconf = higlass.display([
+        View([
+            Track('left-axis'),
+            Track('top-axis'),
+            Track('osm-tiles', position='center'),
```

Comment:
```suggestion
            Track('osm-tiles'),
```

For `osm-tiles` a default position is already defined: https://github.com/higlass/higlass-python/blob/ee09d8f58bf441878c91539475fcde4cee1bbba6/higlass/client.py#L23